nextcord/player.py

```diff
@@ -791,6 +816,16 @@ def _set_source(self, source: AudioSource) -> None:
             self.source = source
             self.resume(update_speaking=False)
 
+    @property
+    def error(self) -> Optional[Exception]:
+        """Optional[:class:`Exception`]: The exception that caused playback to stop, if any."""
+        return self._current_error
+
+    @property
+    def has_failed(self) -> bool:
+        """bool: Whether playback terminated due to an exception."""
+        return self._current_error is not None
+
```

Comment:
Add these in VoiceClient instead, the user has no good way of accessing them,  they also won't be included in the documentation build.